Automattic/jetpack
# Jetpack Social | Warn the user that their scheduled posts won't get shared if they will get over limits.

Part of #34182, This PR fixes the messaging regarding scheduled posts for Jetpack Social to warn the user that their scheduled posts won't get shared if they will get over limits.

This PR is a result of a joint/paring session between me and. @gmjuhasz 

## Proposed changes:

* Add a visual meter bar on post edit page to display the current limit status
* Update the messaging around limits especially for scheduled posts

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Boot up this PR
* Run `jetpack install plugins/social`
* Run `jetpack watch plugins/social`
* Goto post edit and open Jetpack Social sidebar
* Manipulate the selectors in `ackages/publicize-components/src/social-store/selectors/shares-data.ts`
* Test different scenarios to confirm the messaging works as expected

Jetpack Social admin page 
<img width="584" alt="Screenshot 2023-11-17 at 5 00 12 PM" src="https://github.com/Automattic/jetpack/assets/18226415/2a67a359-827b-49a0-b50f-3df5a75ad6e7">

Different scenarios on post editor

<img width="304" alt="Screenshot 2023-11-17 at 4 46 54 PM" src="https://github.com/Automattic/jetpack/assets/18226415/77908447-8352-4222-9e8c-cc97754b0466">
<img width="305" alt="Screenshot 2023-11-17 at 4 46 15 PM" src="https://github.com/Automattic/jetpack/assets/18226415/f93ef322-3b57-4083-a28a-7b46b907603d">
<img width="296" alt="Screenshot 2023-11-17 at 4 47 40 PM" src="https://github.com/Automattic/jetpack/assets/18226415/2127ecf9-6f67-4863-a341-50ce26ef6fd3">
<img width="290" alt="Screenshot 2023-11-17 at 4 48 28 PM" src="https://github.com/Automattic/jetpack/assets/18226415/0d324288-a1d8-4ff6-9b1a-432e6145a1b2">
<img width="293" alt="Screenshot 2023-11-17 at 4 48 56 PM" src="https://github.com/Automattic/jetpack/assets/18226415/df58fb70-5941-4f19-b33d-ab76c85bdf3c">
<img width="289" alt="Screenshot 2023-11-17 at 4 49 34 PM" src="https://github.com/Automattic/jetpack/assets/18226415/e52890d2-a6c7-415a-ac02-7b2dcb7596c5">
<img width="286" alt="Screenshot 2023-11-17 at 4 50 14 PM" src="https://github.com/Automattic/jetpack/assets/18226415/7dc68a04-f748-4cd6-91e4-614891b78a3a">
